build.sbt

```diff
@@ -23,13 +23,13 @@ import uk.gov.hmrc.SbtAutoBuildPlugin
 
 val appName = "vat-designatory-details-frontend"
 
-val govUkFrontendVersion       = "0.65.0-play-26"
-val hmrcUkFrontendVersion      = "0.50.0-play-26"
+val govUkFrontendVersion       = "0.77.0-play-26"
+val hmrcUkFrontendVersion      = "0.76.0-play-26"
 val bootstrapPlayVersion       = "2.3.0"
-val govTemplateVersion         = "5.63.0-play-26"
-val playPartialsVersion        = "7.1.0-play-26"
-val playUiVersion              = "9.0.0-play-26"
-val playLanguageVersion        = "4.10.0-play-26"
+val govTemplateVersion         = "5.68.0-play-26"
```

Comment:
I don't think we need this anymore. 
Would you also mind checking the others as I think we can remove:
bootstrap-play-26
play-test